### PR TITLE
add new demo with list/detail patch feature

### PR DESCRIPTION
**Pull request by frederikhors**

### CODE REVIEW

File: examples/demo1/src/lib/kitql/kitQLClient.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { KitQLClient } from '@kitql/client';
```
Comment (jycouet) [inline examples/demo1/src/lib/kitql/kitQLClient.ts:1]: A graphql thing is to put in graphql folder (at least for the demo)

File: examples/demo1/tests/test.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { expect, test } from '@playwright/test';
```
Comment (jycouet) [inline examples/demo1/tests/test.ts:1]: I guess that it's will fail

**Review by jycouet — CHANGES_REQUESTED** [review]
Hi @frederikhors , 
Thx for the PR, but I have to say that it's quite far from the Demo1 I expect to deliver to users.

Today's demo1 is far to be perfect and outdated, bu has some concept that I what to keep.

Let me know how you want to proceed.

**Comment by frederikhors** [comment]
I'll do it again now.

**Pull request closed without merging**